packages/gcf-utils/test/gcf-bootstrapper.ts

```diff
@@ -68,15 +68,15 @@ describe('GCFBootstrapper', () => {
         .resolves({id: 1234, secret: 'foo', webhookPath: 'bar'});
 
       enqueueTask = sinon.stub(bootstrapper, 'enqueueTask');
-      sinon.stub(bootstrapper, 'getInstallationToken');
-      handler = await bootstrapper.gcf(async app => {
-        app.auth = () =>
-          new Promise<GitHubAPI>(resolve => {
-            resolve(GitHubAPI());
-          });
+      sinon
+        .stub(bootstrapper, 'getAuthenticatedOctokit')
+        .returns(Promise.resolve(new Octokit()));
+      handler = bootstrapper.gcf(async app => {
         app.on('issues', spy);
-        app.on('schedule.repository', spy);
-        app.on('err', sinon.stub().throws());
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        app.on('schedule.repository' as any, spy);
```

Comment:
Octokit is now strict about event names, I've been talking to @gr2m about how we might approach this going forward.